src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/cli/CommandRunnerImporterCommandsTest.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.cli
+
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.verifyNoInteractions
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.ContextConfiguration
+import uk.gov.justice.digital.hmpps.pecs.jpc.TestConfig
+import java.time.LocalDate
+
+@SpringBootTest(args = ["--report-import", "--from=2022-01-13", "--to=2022-01-14"], webEnvironment = WebEnvironment.NONE)
+@ActiveProfiles("test")
+@ContextConfiguration(classes = [TestConfig::class])
+class CommandRunnerImporterCommandsTest {
```

Comment:
Two separate tests are needed for testing the commands when running as if the application was genuinely running due to the way the arguments for the test (i.e. identifying the command to run) are wired in at the class declaration, you cannot override at the individual test level.